### PR TITLE
[FIX]: adding json header

### PR DESCRIPTION
**Pull request by bengriffin1**
### 📦 Pull Request

Adding a 'Content-Type' header to fix broken API calls to Fortmatic API.

### CODE REVIEW

File: src/utils/rest.ts [hunk 1]
```diff
@@ -45,10 +49,13 @@ export function post<TBody extends Record<string, string | number | boolean> = {
   url: string,
   secretApiKey: string,
   body: TBody,
+  addJsonHeader = false,
```
Comment (justinnout) [inline src/utils/rest.ts:52]: Can this be updated to pass in custom headers?

Comment (bengriffin1) [reply src/utils/rest.ts]: Yep good call, updated in latest

**Review by justinnout — APPROVED** [review]
Love it

**Comment by magiclabsFE** [comment]
:rocket: PR was released in [`v1.9.2`](https://github.com/magiclabs/magic-admin-js/releases/tag/v1.9.2) :rocket:

**Pull request merged**